### ISSUE TITLE
Suggestion for improvement

### ISSUE THREAD

**Issue by Martin-BW** [tmjo/charger-card#3]
- cableLocked: It shows "off" when the cable is locked. It would be better if it would show "on" in this case.
- The led's aren't blue if smart charging is on
- The tooltip for DynamicCircuitLimit shows "Dyn Charger Limit"
- Power should show one decimal

**Comment by tmjo** [comment]
Great inputs, I'll look at it later today. Can you doublecheck the cableLocked? Mine are showing correct. Both names were changed in the breaking change, can it be that they somehow have the old names in your case?

It is a very suboptimal solution the way I handle sensor names today, I kindof just use the main sensor and find the "basename" and then use the default names from integration. Will look into a better way to do it, or at least a possibility to override it.

**Comment by Martin-BW** [comment]
CableLocked: Shows "off" if cable is locked, shows "on" if the cable is unlocked.

In my opinion it wuold be better if it shows "on" if the cable is locked.

**Comment by tmjo** [comment]
Aha, I see you're right. Mine shows "on" and "unlocked" too. But that comes from the integration itself, I'll raise the question there and see why this is. If you see in dev tools you will probably see the same as me, that it is not directly related to the card. Of course I can change it, but it seems a bit dirty.

![image](https://user-images.githubusercontent.com/54450177/111201465-1907f180-85c3-11eb-9cef-37a8bf6afc2d.png)

**Comment by tmjo** [comment]
Please test card v.0.0.5 for the other stuff (didn't do anything about the cable lock for now).

**Comment by tmjo** [comment]
About the lock; I discussed it briefly in the integration team, although it beats me why and we also found it confusing, it seems like it is intended by HA. See https://www.home-assistant.io/integrations/binary_sensor/

This detail:
![image](https://user-images.githubusercontent.com/54450177/111205326-6ab27b00-85c7-11eb-8dcb-cb7e616cd4ec.png)

We may change it in the integration to be a lock, or remove the device_class: lock later though. I'll leave it and see for now.
